### DISCUSSION TITLE
Limit the distance of the zone-wide teleport for the Hellfire PVP event, or remove it.

### DISCUSSION THREAD

**Discussion by Zelfie** [chromiecraft/chromiecraft#5878] [Suggestions]
World boss fun is going on, and if anyone ignores their chat and focuses on the group content at hand after a wipe, BOOM you've been teleported half the zone away because a PVP event we've all outgrown is popping off.
World PVP events were fun when the content was relevant and we were all running around, but now, especially in Hellfire, it's just interrupting actual content.
Shrink the size of the teleport, or just remove it please.
Numbers wise, are people actively playing this content? I'm somewhat doubtful.

**Comment by 55Honey** [comment]
How does ignoring your chat affect a world boss fight?
You need to actively sign up for PvP to be teleported. Unless you take action yourself and join the queue manually with a command, you're not affected.

**Reply by Zelfie** [reply]
While doing the World boss, wiping, getting pvp grieved, the whole raid then got teleported around by the hellfire event, not noticing the 30 second demand that it'll teleport us because we weren't looking at chat.
No-one had signed up for the event, we were doing World bosses. Is there any way it auto-involves people? 20 people suddenly got teleported from the world boss to the centre of hellfire

**Reply by 55Honey** [reply]
The PvP event requires a manual signup per player. It checks a list of players which have done so. That list is not accessible by any other script or part of the core. It teleports only listed players. I currently don't see a way in which the PvP script could fail and summon too many players when interacting with the worldbosses.

**Reply by Zelfie** [reply]
Dunno what to say Honey.
We had a wipe, and 15 odd people got teleported despite not having signed up for the Hellfire event because we were busy playing with the world boss.
